mobile/native/src/api.rs

```diff
@@ -848,3 +851,12 @@ pub fn delete_dlc_channel(dlc_channel_id: String) -> Result<()> {
 pub fn sync_for_addresses(gap: u32) -> Result<bool> {
     ln_dlc::sync_for_addresses(gap)
 }
+
+pub fn get_new_random_name() -> SyncReturn<String> {
+    SyncReturn(crate::names::get_new_name())
+}
+
+#[tokio::main(flavor = "current_thread")]
+pub async fn update_nickname(nickname: String) -> Result<()> {
```

Comment:
❓ On the topic of the nullable nickname, I think FRB should be able to accept an `Option<String>` here, right?